src/Provider/AbstractProvider.php

```diff
@@ -295,24 +277,16 @@ public function getAccessToken($grant = 'authorization_code', array $params = []
 
     /**
      * Get an authenticated request instance.
-     *
-     * Creates a PSR-7 compatible request instance that can be modified.
-     * Often used to create calls against an API that requires authentication.
-     *
-     * @param  string $method
-     * @param  string $url
-     * @param  AccessToken $token
-     * @return RequestInterface
      */
-    public function getAuthenticatedRequest($method, $url, AccessToken $token)
+    public function getRequest($method, $url, AccessToken $token, $body = null)
```

Comment:
Figured a provider request might not be authorized if the access token is null, or if it's considered to always be authorized then why not call it `getRequest` to be consistent with `getResponse`.